### ISSUE TITLE
No analysis on wallet interaction patterns and will likely lead to a lot of false positives

### ISSUE THREAD

**Issue by SilentEyes343** [analysiooor/clusters#7]
just poor execution in general 

No analysis on wallet interaction patterns was done at all

just trying to reach for any type of connection between wallets that bridged between each other using the reroute feature in cross chain bridges 

even if that funding was one way (sent to a wallet but never received back for example)

too many instances for false positives 

report is also not conclusive

**Comment by Mooun13** [comment]
My wallet is also in one of his clusters and I fully agree you here! Hopefully layerzero team is doing a good job by filtering out such reports...would be a mess.

**Comment by Tinycl** [comment]
the only logic detecting a sybil is  interaction bridge for example  lifi ,  does not consider the other inertactions with other dapp.

**Comment by Mooun13** [comment]
Yes, this is definetly not a reliable strategy to detect sybils. Pretty sure his 246 reports will be deleted. But it's still a bit scary to see my own wallet flagged...I just hope l0 team does well cleaning such reports.
